modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/transformer/test/JournalTransformerTest.java

```diff
@@ -409,6 +404,9 @@ private void _testTransformDDMFormField(
 	@Inject
 	private Portal _portal;
 
+	@Inject(filter = "component.name=*.RegexTransformerListener")
```

Comment:
@JanisZhang Could you use the full class name here?

Reply:
Hi @dantewang, force-pushed. thx